introduction.md

```diff
@@ -0,0 +1 @@
+Hello I am here to teach about Git Hub.
```

Comment:
I enjoy all the Paths I have finished on Code School, but JavaScript taught me the most!.